BUILD

```diff
@@ -1969,5 +1968,3 @@ grpc_cc_library(
         "//src/proto/grpc/core:stats_proto",
     ],
 )
-
-grpc_generate_one_off_targets()
```

Comment:
No, we need to keep this, to match what we have internally. Just make it a "pass" function here.

Reply:
Done.